### PR TITLE
EddieWang.Sia-UI version 1.5.7

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/33441)

### CODE REVIEW

**Comment by Xanthus58** [comment]
image



Works perfectly locally, not sure what the issue is exactly.

File: manifests/e/EddieWang/Sia-UI/1.5.7/EddieWang.Sia-UI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: EddieWang.Sia-UI
+PackageVersion: 1.5.7
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/e/EddieWang/Sia-UI/1.5.7/EddieWang.Sia-UI.installer.yaml:7]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/e/EddieWang/Sia-UI/1.5.7/EddieWang.Sia-UI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+License: Copyright © 2021 Eddie Wang
+ShortDescription: Sia is the leading decentralized cloud storage platform.
+Description: Sia is the leading decentralized cloud storage platform. No signups, no servers, no trusted third parties. Sia leverages blockchain technology to create a data storage marketplace that is more robust and more affordable than traditional cloud storage providers.
+Moniker: Sia
+Tags:
+- crypto
+- cloud
+- storage
+- decentralized
+- Open Source
```
Comment (ItzLevvie) [inline manifests/e/EddieWang/Sia-UI/1.5.7/EddieWang.Sia-UI.locale.en-US.yaml:20]:
```suggestion
- open-source
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Xanthus58** [comment]
Thanks lev; really appreciate all the help :)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**